app/src/main/res/values/strings.xml

```diff
@@ -76,6 +76,12 @@
     <!-- Content description for close button used in "contextual feature recommendation" (CFR) popups -->
     <string name="cfr_dismiss_button_default_content_description">Dismiss</string>
 
+    <!-- Total cookie protection "contextual feature recommendation" (CFR) -->
+    <!-- Text for the main message. -->
+    <string name="tcp_cfr_message">Our most powerful privacy feature yet isolates cross-site trackers.</string>
+    <!-- Text displayed that links to website containing documentation about the "Total cookie protection" feature. -->
+    <string name="tcp_cfr_learn_more">Learn about Total Cookie Protection</string>
```

Comment:
Cannot find a design for the feature, but just as a though, may be a link text should be with \<u> tags

Reply:
Found the design, it is already underlined by `textDecoration = TextDecoration.Underline`. Sorry for raising extra attention.